src/app/page.tsx

```diff
@@ -1,12 +1,71 @@
-"use client"
-import "./style.css"
+"use client";
+import "./style.css";
 
 export default function Home() {
   return (
-    <button onClick={function(){
-      alert("Olá mundo!")
-    }}>
-      Clique aqui!
-    </button>
+    <div className="navigation-bar">
+      <div>
+        <svg
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <circle cx="11" cy="11" r="7" stroke="#33363F" stroke-width="2" />
+          <path
+            d="M20 20L17 17"
+            stroke="#33363F"
+            stroke-width="2"
+            stroke-linecap="round"
+          />
+        </svg>
+      </div>
```

Comment:
Vamos adicionar botões em volta desses svgs pra sinalizar que eles devem ter ações. Por mais que alguns possas se tornar links mais pra frente, pelo menos a gente tem um indicativo em html que esses elementos tem interação com o usuário.

```suggestion
      <div>
      <button>
        <svg
          width="24"
          height="24"
          viewBox="0 0 24 24"
          fill="none"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle cx="11" cy="11" r="7" stroke="#33363F" stroke-width="2" />
          <path
            d="M20 20L17 17"
            stroke="#33363F"
            stroke-width="2"
            stroke-linecap="round"
          />
        </svg>
        </button>
      </div>
```